html/js/vr-party-participant.js

```diff
@@ -165,7 +175,20 @@ function viewersApplyState() {
     }
 
     if (_model_state.isolate_id !== undefined) {
-        viewersApply('isolate', _model_state.isolate_id);
+        var ids = _model_state.isolate_id;
+        if ((LMV_VIEWER_VERSION === '1.2.13' || LMV_VIEWER_VERSION === '1.2.14') &&
+            ids.length > 0 && typeof ids[0] === 'number') {
+                // getNodesByIds can throw an exception when the model isn't sufficiently loaded
+                // Catch it and try to apply the viewer state again in a second
+                 try {
+                    ids = _viewerLeft.model.getNodesByIds(ids);
+                }
+                catch (ex) {
+                    setTimeout(function() { viewersApplyState(); }, 1000);

```

Comment:
well, it is really a retry and if the model is not ready yet. therefore I think the timeout is good.